src/stream/src/executor/hash_agg.rs

```diff
@@ -41,6 +40,9 @@ use crate::executor::aggregation::{
 use crate::executor::error::StreamExecutorError;
 use crate::executor::monitor::StreamingMetrics;
 use crate::executor::{BoxedMessageStream, Message, PkIndices, PROCESSING_WINDOW_SIZE};
+use crate::task::{EvictableHashMap, ExecutorCache, LruManagerRef};
+
+type AggStateMap<K, S> = ExecutorCache<K, Option<Box<AggState<S>>>, PrecomputedBuildHasher>;
```

Comment:
Why use `PrecomputedBuildHasher`? It's somehow a very hacky thing which uses the 8-byte prefix of input as the hash value of it. Are you sure you want this?

Reply:
Oh you are right! The hash value is precomputed in `HashKey::build`. My intention was to get improve the performance. However, the hash code is generated from  `get_hash_values`, which is also used in shuffle, so `PrecomputedBuildHasher` may cause hot keys in operator hash map.